rasa_utils/bot_server_channel.py

```diff
@@ -98,95 +98,102 @@ def __init__(
+
+        @custom_webhook.route("/conversations/<cid>/say", methods=["GET"])
+        def say(cid):
+            message = bytes(request.args.get("message"), "utf8")
+            _payload = bytes(request.args.get("payload", ""), "utf8")
+            _display_name = bytes(request.args.get("display_name", ""), "utf8")
+            _uuid = bytes(request.args.get("uuid", ""), "utf8")
+            logger.info(message)
+
+            if len(_display_name) > 0:
+                display_name, = _display_name
+                tracker = self.agent.tracker_store.get_or_create_tracker(cid)
+                if (
+                    "display_name" in tracker.current_slot_values()
+                    and tracker.get_slot("display_name") != display_name
+                ):
+                    tracker.update(SlotSet("display_name", display_name.decode("utf-8")))
+                    self.agent.tracker_store.save(tracker)
+
+            if message == "_restart":
+                self.message_store.clear(cid)
+            else:
+                if len(_uuid) > 0:
+                    self.message_store.log(
+                        cid,
+                        cid,
+                        {"type": "text", "text": message.decode("utf-8")},
+                        _uuid.decode("utf-8"),
+                    )
+            if len(_payload) > 0:
+                on_new_message(
+                    UserMessage(
+                        _payload[0].decode("utf-8"),
+                        output_channel=BotServerOutputChannel(self.message_store),
+                        sender_id=cid,
+                    ),
+                    preprocessor=self.preprocessor
                 )
-            )
-        else:
-            self.on_message(
-                UserMessage(
-                    message.decode("utf-8"),
-                    output_channel=BotServerOutputChannel(self.message_store),
-                    sender_id=cid,
+            else:
+                on_new_message(
+                    UserMessage(
+                        message.decode("utf-8"),
+                        output_channel=BotServerOutputChannel(self.message_store),
+                        sender_id=cid,
+                    ),
+                    preprocessor=self.preprocessor
                 )
-            )
-
-    @app.route("/health", methods=["GET"])
-    def health(self, request):
-        return "healthy"
-
-    @app.route("/", branch=True, methods=["GET"])
-    @check_cors
-    def static(self, request):
-        if self.static_files is None:
-            return NoResource()
-        else:
-            return File(self.static_files)
-
-    def start(self, on_message):
-        self.on_message = on_message
-        logger.info("Started http server on port %d" % self.port)
-        self.app.run("0.0.0.0", self.port)
-
-    def start_async_listening(self, message_queue):
-        self.start(message_queue.enqueue)
+            return "OK"
```

Comment:
I'd prefer to return a proper Flask status code here: 
`return make_response("OK", 200)`

`from flask import make_response`

Reply:
Sure, I'll add this in.